### PR TITLE
5357 Add window position config keys for Clipboard & main window.

### PR DESCRIPTION
**Pull request by sam-m888**
Please test on:
- [ ] Apple Mac @jralls
- [ ] MS-Windows @bpisoj 

Tested on
- [x] Linux (Ubuntu 14.04.3)

Any feedback welcomed.

https://gramps-project.org/bugs/view.php?id=5357

### CODE REVIEW

**Comment by Nick-Hall** [comment]
Applications normally leave the positioning of windows to the window manager.  We should investigate setting some window manager hints instead.

File: gramps/gen/config.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #
 
 """
-This package implements access to GRAMPS configuration.
+This package implements access to Gramps configuration.
 """
```
Comment (jralls) [inline gramps/gen/config.py]: Put editorial changes like this in a separate commit.

**Comment by jralls** [comment]
Rather than hard-coding values like this we should be recording the window sizes and positions at the end of the session and restoring them when the program is next launched.

File: gramps/gen/config.py (hunk 1 above)
Comment (sam-m888) [inline gramps/gen/config.py]: Ok removed the editorial changes and pushed back to github and it closed !
https://github.com/sam-m888/gramps/commit/28f3d90f

**Comment by sam-m888** [comment]
@jralls  Ok I'll give that a try.

**Comment by sam-m888** [comment]
@Nick-Hall For hints do you mean like how some of the dialogs automatically center via

* Gtk.WindowPosition.CENTER_ON_PARENT
 and maybe for the main window:
* Gtk.WindowPosition.CENTER
